checkstyle-sonar-plugin/src/main/resources/org/sonar/plugins/checkstyle/rules.xml

```diff
@@ -1245,6 +1245,10 @@
     <name><![CDATA[Package Declaration]]></name>
     <configKey><![CDATA[Checker/TreeWalker/PackageDeclaration]]></configKey>
     <status>READY</status>
+    <cardinality>MULTIPLE</cardinality>
```

Comment:
cardinality shouldn't be multiple.
We only have 1 property that is either true/false for this check. Users won't activate 2 instances with both values on.